features/step_definitions/custom_web_steps.rb

```diff
@@ -142,6 +142,15 @@ def take_screenshots_with_login
   page.driver.browser.switch_to.alert.dismiss
 end
 
+And /^I should not see any alert$/ do
+  begin
+    page.driver.browser.switch_to.alert
+  rescue Selenium::WebDriver::Error::NoAlertPresentError
+    true
+  end
+  false
+end
```

Comment:
I'm not really proud with this solution but this is the only way I found to check if an alert pops up on internet. If someone has a more elegant solution, let he speak now or or forever hold his piece! :D